src/dotnet/APIView/APIViewWeb/FilenameSafeStringParser.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace APIViewWeb
+{
+    public class FilenameSafeStringParser
+    {
+        public static string ParseLanguageName(string languageName)
+        {
+            switch(languageName.ToLower())
+            {
+                case "c#":
+                    return "csharp";
+                case "c++":
+                    return "cplusplus";
+                default:
+                    return languageName.ToLower();
+            }
+        }
+    }
+}
```

Comment:
@praveenkuttappan Please can you give your expert C# guidance on this? It is trying to solve the problem that the C# and C++ language parsers return "C#" and "C++", and this doesn't play nice in terms of using those strings as values in CSS or filename references. Rather than duplicate this code in a few different places, I suggested a utility function, although I don't know if 1) there is a better way to do this in C#, and 2) where this utility function should go (I'm guessing its current location is less than ideal). Thanks!

Reply:
I think another better place to add this is in ReviewModel itself to have a method called GetLanguageCssSafeName(). @Vailorium Please make sure to add a comment why is this method required so anyone who check the code in future will  have a clear idea on why we need this method